test/to_string_test.hpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+
+using namespace std;
+
+void toStringTest() {
```

Comment:
https://google.github.io/styleguide/cppguide.html#Function_Names

```suggestion
void ToStringTest() {
```